openshift/e2e-common.sh

```diff
@@ -28,7 +28,8 @@ readonly OLM_NAMESPACE="openshift-marketplace"
 # if you want to setup the nightly serving/eventing, set `release-next` OR
 # set release branch name for example: release-v0.19.1
 readonly SERVING_BRANCH="release-next"
-readonly EVENTING_BRANCH="release-next"
+# Pin the version due to issue in current nightly version
+readonly EVENTING_BRANCH="release-v0.19.2"
```

Comment:
Since the fix is merged in Eventing repo we can go back to nightly version as well.

Reply:
@dsimansk : +1